.github/workflows/test.yml

```diff
@@ -0,0 +1,355 @@
+name: Test all packages
+
+on:
+  pull_request:
+    types:
+      - synchronize
+  push:
+    branches:
+      - '**'
+
+jobs:
```

Comment:
I wonder if we can cache the `npm i` as mentioned [here](https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action) for all below jobs. I'm curios how much time we can safe 🤔

Reply:
I think the `setup-node` action we're using already takes care of that:

> The action has a built-in functionality for caching and restoring dependencies. It uses actions/cache under hood for caching dependencies but requires less configuration settings. Supported package managers are npm, yarn, pnpm (v6.10+). The cache input is optional, and caching is turned off by default.

https://github.com/actions/setup-node#caching-packages-dependencies

Reply:
Thanks for checking that 🎖️